source/_layouts/base.twig

```diff
@@ -25,6 +25,24 @@
 
     <link rel="preconnect" href="https://ajax.googleapis.com" pr="1.0" />
     <link rel="preconnect" href="https://ssl.google-analytics.com" pr="0.8" />
+
+    <meta property="og:url" content="{{ ('https://www.php-fig.org/' ~ page.url) }}" />
```

Comment:
do we have some parameters for the deploy url?

Reply:
Twitter wants the image URL to be absolute.... this should be something provided by plafrorm sh i guess...

Reply:
I see that you removed it. Probably the URL itself of the page being linked is enough, we don't need to care about permanent URLs due to the fact that the site is static.

Reply:
Nope, I tested it, the URL has to be absolute for sure. Worst case, we could use the official domain.

Reply:
PSH gives us the variables in the build here: https://docs.platform.sh/development/variables.html#platformsh-provided-variables

I don't know if it's worth the hassle for something that we do not change basically ever...